kconfigs/qemu.conf

```diff
@@ -1,5 +1,6 @@
 CONFIG_TEE=y
 CONFIG_OPTEE=y
+CONFIG_OPTEE_BENCHMARK=y
```

Comment:
Same comment here, a manual Kconfig change is what I'd prefer ... alternatively, you create a another *.conf file, that you merge with qemu.conf if `CFG_TEE_BENCHMARK=y`, something like [this](https://github.com/OP-TEE/build/blob/ba038a5e498fd2587ef1418842f06a598323663c/juno.mk#L73-L80).